subsys/zigbee/Kconfig

```diff
@@ -156,33 +156,44 @@ config ZBOSS_TRACE_LOG_LEVEL
 config ZBOSS_TRACE_MASK
 	hex "Trace mask of ZBOSS stack logs"
 	default 0x0000
-	range 0x0000 0xFFFF
+	range 0x00000000 0xFFFFFFFF
 	help
 	  Selectively enable Zigbee binary trace logs.
 	  The mask value should be a bitwise OR of values assigned to selected modules.
 
 	  Available modules are:
 
-	  - 0x8000 SPI platform implementation
-	  - 0x4000 Zigbee Green Power
-	  - 0x2000 Custom components
-	  - 0x1000 UART and NCP transport
-	  - 0x0800 Application
-	  - 0x0400 MAC Lower Layer
-	  - 0x0200 Zigbee Light Link
-	  - 0x0100 Zigbee Cluster Library
-	  - 0x0080 Security
-	  - 0x0040 Zigbee Device Object
-	  - 0x0020 Zigbee Smart Energy
-	  - 0x0010 Application Support layer
-	  - 0x0008 Network layer
-	  - 0x0004 MAC layer
-	  - 0x0002 Memory management
-	  - 0x0001 Common
-
-	  For example, in order to enable traces related to OTA DFU, one should set this option to 0x4100.
-
-	  Note: For general debugging purposes, please use 0x0C48.
+	  - 0x04000000 Reserved
+	  - 0x02000000 Reserved
+	  - 0x01000000 Reserved
+	  - 0x00800000 Reserved
+	  - 0x00400000 Reserved
+	  - 0x00200000 Zigbee NCP: transport (LL)
+	  - 0x00100000 Zigbee NCP: command adapters (HL)
+	  - 0x00080000 ZCL: Over The Air firmware upgrade
```

Comment:
Shouldn't this be FOTA?
`ZCL: Firmware over-the-air upgrade`?

Reply:
I agree. It is better to keep it consistent and change to FOTA.